core/modules/qmeta/qmetaPythonWrapper.cc

```diff
@@ -0,0 +1,107 @@
+        .value("SYNC", QInfo::QType::SYNC)
+        .value("ASYNC", QInfo::QType::ASYNC)
+        .value("ANY", QInfo::QType::ANY)
+        .export_values()
+        ;
+
+    py::enum_<QInfo::QStatus>(clsQInfo, "QStatus")
+        .value("EXECUTING", QInfo::QStatus::EXECUTING)
+        .value("COMPLETED", QInfo::QStatus::COMPLETED)
+        .value("FAILED", QInfo::QStatus::FAILED)
+        .value("ABORTED", QInfo::QStatus::ABORTED)
+        .export_values()
+        ;
+
+    clsQInfo
+        .def(py::init<>())
+        .def(py::init<QInfo::QType, CzarId, std::string const&,
+                std::string const&, std::string const&,
+                std::string const&, std::string const&,
+                std::string const&, std::string const&,
+                QInfo::QStatus, std::time_t, std::time_t, std::time_t>(),
+                "qType"_a, "czarId"_a, "user"_a, "qText"_a, "qTemplate"_a,
+                "qMerge"_a, "qProxyOrderBy"_a, "resultLoc"_a, "msgTableName"_a,
+                "qStatus"_a=QInfo::EXECUTING, "submitted"_a=std::time_t(0),
+                "completed"_a=std::time_t(0), "returned"_a=std::time_t())
+        .def("queryType", &QInfo::queryType)
+        .def("queryStatus", &QInfo::queryStatus)
+        .def("czarId", &QInfo::czarId)
+        .def("user", &QInfo::user)
+        .def("queryText", &QInfo::queryText)
+        .def("queryTemplate", &QInfo::queryTemplate)
+        .def("mergeQuery", &QInfo::mergeQuery)
+        .def("proxyOrderBy", &QInfo::proxyOrderBy)
+        .def("resultLocation", &QInfo::resultLocation)
+        .def("msgTableName", &QInfo::msgTableName)
+        .def("submitted", &QInfo::submitted)
+        .def("completed", &QInfo::completed)
+        .def("returned", &QInfo::returned)
+        .def("duration", &QInfo::duration)
+        ;
+
+    py::class_<QMeta, std::shared_ptr<QMeta>>(mod, "QMeta")
+        .def_static("createFromConfig", &QMeta::createFromConfig)
+        .def("getCzarID", &QMeta::getCzarID)
+        .def("registerCzar", &QMeta::registerCzar)
+        .def("setCzarActive", &QMeta::setCzarActive)
+        .def("registerQuery", &QMeta::registerQuery)
+        .def("addChunks", &QMeta::addChunks)
+        .def("assignChunk", &QMeta::assignChunk)
+        .def("finishChunk", &QMeta::finishChunk)
+        .def("completeQuery", &QMeta::completeQuery)
+        .def("finishQuery", &QMeta::finishQuery)
+        .def("findQueries", &QMeta::findQueries,
+                "czarId"_a=0, "qType"_a=QInfo::ANY, "user"_a=std::string(),
+                "status"_a=std::vector<QInfo::QStatus>(),
+                "completed"_a=-1, "returned"_a=-1)
+        .def("getPendingQueries", &QMeta::getPendingQueries)
+        .def("getQueryInfo", &QMeta::getQueryInfo)
+        .def("getQueriesForDb", &QMeta::getQueriesForDb)
+        .def("getQueriesForTable", &QMeta::getQueriesForTable)
+        ;
+
+    // Exception classes
+    static py::exception<Exception> Exception(mod, "Exception");
```

Comment:
Are you sure you want to use the name of the built-in `Exception`? What if the user does an `import *`?